benchmark_runner/grafana/perf/jsonnet/jsonnetfile.lock.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Rather than `.lock`, I suggest `.version` if that's possible (i. e. if jsonnet doesn't have some kind of hard-coded requirement for that filename).  `lock` implies that it's locking against some kind of concurrency issue; this is about versioning, not concurrency.

Reply:
That looks fine.